<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="Age_Entry">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Name_Entry">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Country_Code_Label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Age:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Name_Label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_Player_Button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Player</string>
   </property>
  </widget>
  <widget class="QLabel" name="Gender_Label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gender:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Role_Label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Role:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Gender_ComboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Male</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Female</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="Role_ComboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>All-Rounder</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Batsman</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bowler</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
